src/app/gallery/page.tsx

```diff
@@ -1,5 +1,95 @@
-export default function page() {
+"use client";
+import Image from "next/image";
+import Pagination from "../../../components/pagination/Pagination";
+import ArtworkCard from "../../../components/ArtworkCard";
+import gallery_image from "../../../public/gallery/gallery_image.svg";
+// eslint-disable-next-line
```

Comment:
Please make these comments more specific. `// eslint-disable-next-line react-hooks/rules-of-hooks` should work